examples/with-docker/apps/api/Dockerfile

```diff
@@ -20,14 +20,11 @@ RUN apk add --no-cache libc6-compat
 WORKDIR /app
 
 # First install dependencies (as they change less often)
-COPY .gitignore .gitignore
```

Comment:
- This file is not required for `yarn install`
- This file is included in `/app/out/full` for build step, therefore, no need to add here